config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-periodics-main-upgrades.yaml

```diff
@@ -86,9 +86,9 @@ periodics:
           cpu: 7300m
   annotations:
     testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-azure
-    testgrid-tab-name: capi-periodic-upgrade-main-1-23-1-24
+    testgrid-tab-name: capi-periodic-upgrade-main-1-24-1-25
 
-- name: periodic-cluster-api-provider-azure-e2e-workload-upgrade-1-24-1-25-main
+- name: periodic-cluster-api-provider-azure-e2e-workload-upgrade-1-25-1-26-main
```

Comment:
side note: should we should bump the kubekins image to 1.26 for all periodic jobs from main?

https://github.com/kubernetes/test-infra/commit/f7da6f4dd0fc5b68fd01eabb52fda8f41349a054